tests/kolainst/destructive/auto-prune.sh

```diff
@@ -71,11 +100,11 @@ bootloader_orig=$(sha256sum /boot/loader/entries/*)
 # now try to deploy a third deployment without early pruning; we should hit ENOSPC
 consume_bootfs_space
 rpm-ostree rebase :modkernel2
-if OSTREE_SYSROOT_OPTS=no-early-prune ostree admin finalize-staged |& tee out.txt; then
+cursor=$(journal_cursor)
+if OSTREE_SYSROOT_OPTS=no-early-prune ostree admin finalize-staged; then
     assert_not_reached "successfully wrote kernel without auto-pruning"
 fi
-assert_file_has_content out.txt "No space left on device"
-rm out.txt
+assert_journal_grep "$cursor" "updating bootloader in two steps"
```

Comment:
I think this one needs to be e.g.

```bash
rm -vf err.txt
if OSTREE_SYSROOT_OPTS=no-early-prune ostree admin finalize-staged 2>err.txt; then
    assert_not_reached "successfully wrote kernel without auto-pruning"
fi
assert_file_has_content err.txt "No space left on device"
```

?

Reply:
I did that in the last push right?

Reply:
But yeah I still haven't re-setup cosa, I've just been playing the game of "look at the CI result, blindly update the test and push, go do other things" rinse repeat...

Reply:
The key change was using `journalctl -t ostree` to filter on just output from that binary and avoid the [grep finding itself](https://github.com/ostreedev/ostree/pull/3285#issuecomment-2292401986) problem.

Reply:
Hmm, in the diff I'm seeing, this is changing the check we're doing on this section of the test. Before we were checking for "No space left on device" (which would show up on stderr), now we're checking for "updating bootloader in two steps" in the journal.

> The key change was using `journalctl -t ostree` to filter on just output from that binary and avoid the [grep finding itself](https://github.com/ostreedev/ostree/pull/3285#issuecomment-2292401986) problem.

Yeah... I walked right into that one too.

Reply:
You're right, I have fixed that thanks!